### DISCUSSION TITLE
[RelayCommand] Not woking when method name end with 'Async'

### DISCUSSION THREAD

**Discussion by JaehwanMango** [CommunityToolkit/dotnet#459] [Q&A] [answered]
Hi, I have Question
=> [RelayCommand] Not woking when method name end with 'Async'

Example
```
// Page.xaml
<Button Command="{Binding = exampleAsync}"

// ViewModel
[RelayCommand]
private async Task exampleAsync()
{
 ...
}
```
exampleAsync() is not work when click the button If method(Command) name end with 'Async'

**Comment by Sergio0694** [comment, marked as answer]
You're binding to the wrong thing. If your method is named `FooAsync`, the command name will be `FooCommand`, and you need to bind to that. [See the docs here](https://learn.microsoft.com/dotnet/communitytoolkit/mvvm/generators/relaycommand) 🙂

**Reply by JaehwanMango** [reply]
Hi, Sergio0694
Thank you so much!
